pkg/cmd/workspace/code.go

```diff
@@ -70,12 +71,16 @@ var CodeCmd = &cobra.Command{
 			workspaceId = *workspace.Id
 		}
 
-		// Todo: make project_select_prompt view for 0 args
 		if len(args) == 0 || len(args) == 1 {
-			projectName, err = util.GetFirstWorkspaceProjectName(workspaceId, projectName, &activeProfile)
+			var selectedProject *string
```

Comment:
No need to declare a variable like this. Just declare it inline below with `:=`.